students/16th/wanie-kim/js/login.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const button = document.getElementById("button");
+const inputId = document.getElementById("id");
+const inputPassword = document.getElementById("password");
+let buttonOff = button.disabled;
+
+inputId.addEventListener("input", activeButton)
+inputPassword.addEventListener("input", activeButton)
+
+function activeButton() {
+    const id = inputId.value;
+    const password = inputPassword.value;
+    buttonOff = (!id || !password) ? true : false;
```

Comment:
와 승완님! 👏🏻
삼항연산자 및 논리연산자 잘 활용하셨네요! 
아래와 같이 작성하면 좀 더 직관적일 것 같아요! 
```suggestion
    buttonOff = (id && password) ? false : true;
```